Sources/Private/MainThread/LayerContainers/Utility/LayerTransformNode.swift

```diff
@@ -18,13 +18,17 @@ final class LayerTransformProperties: NodePropertyMap, KeypathSearchable {
   init(transform: Transform) {
     anchor = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.anchorPoint.keyframes))
     scale = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.scale.keyframes))
-    rotation = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.rotation.keyframes))
+    rotationX = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.rotationX.keyframes))
+    rotationY = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.rotationY.keyframes))
+    rotationZ = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.rotationZ.keyframes))
     opacity = NodeProperty(provider: KeyframeInterpolator(keyframes: transform.opacity.keyframes))
 
     var propertyMap: [String: AnyNodeProperty] = [
       "Anchor Point" : anchor,
       "Scale" : scale,
-      "Rotation" : rotation,
```

Comment:
Removing support for `"*.Rotation"` animation key paths would be a breaking change for anyone relying on this functionality. Is there a way to continue supporting `Rotation` here as well as X/Y/Z?

Reply:
You can see in [Tests/__Snapshots__/AnimationKeypathTests/testMainThreadEngineKeypathLogging.Switch.txt](https://github.com/airbnb/lottie-ios/pull/1930/files#diff-5bd94bedf0fc0d5657b62b8eb004d4c4b8fc57257dc302ae516535cecf4c4d13) how this is part of the public API and how it would cause a breaking change

Reply:
Yes. I can bring it back. I wasn't sure how this was used

Reply:
Done